app/i18n/en/sub.php

```diff
@@ -14,7 +14,9 @@
 		'_' => 'Category',
 		'add' => 'Add a category',
 		'empty' => 'Empty category',
+		'informations' => 'Information',
```

Comment:
I believe it's all supposed to be English? ;-)

Reply:
I know! I just copied it from the feed section.

Reply:
I've changed everything to information!